docs/Makefile

```diff
@@ -16,4 +16,11 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
\ No newline at end of file
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+# This will build the docs and launch a server. To access remotely you can use
+# SSH port forwarding:
+#     ssh -L 8000:localhost:8000 HOST
+# where HOST is the server actually hosting the docs
+server: html
```

Comment:
Minor nit: `server` should be declared a phony target.

Reply:
fixed